source/get-started/next-steps.txt

```diff
@@ -0,0 +1,17 @@
+.. _java-rs-get-started-next-steps:
+
+==========
+Next Steps
+==========
+
+Congratulations on completing the tutorial!
+
+In this tutorial, you created a Java Reactive Strings application that
+connects to a MongoDB deployment hosted on MongoDB Atlas
+and retrieves a document that matches a query. 
+
+Learn more about {+driver-short+} from the following resources:
```

Comment:
Comment this line out until we have some pages to link to, or alternatively link to the existing read/write pages for now